ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { captureException } from '@sentry/browser';
+
+import { Text } from '../../../component-library';
+import { Severity } from '../../../../helpers/constants/design-system';
+import { I18nContext } from '../../../../contexts/i18n';
+
+import {
+  BlockaidReason,
+  BlockaidResultType,
+  SecurityProvider,
+} from '../../../../../shared/constants/security-provider';
+import SecurityProviderBannerAlert from '../security-provider-banner-alert';
+
+/** Reason to description translation key mapping. Grouped by translations. */
+const REASON_TO_DESCRIPTION_TKEY = Object.freeze({
+  [BlockaidReason.approvalFarming]: 'blockaidDescriptionApproveFarming',
+  [BlockaidReason.permitFarming]: 'blockaidDescriptionApproveFarming',
+  [BlockaidReason.setApprovalForAll]: 'blockaidDescriptionApproveFarming',
+
+  [BlockaidReason.blurFarming]: 'blockaidDescriptionBlurFarming',
+
+  [BlockaidReason.seaportFarming]: 'blockaidDescriptionSeaportFarming',
+
+  [BlockaidReason.maliciousDomain]: 'blockaidDescriptionMaliciousDomain',
+
+  [BlockaidReason.signatureFarming]: 'blockaidDescriptionMightLoseAssets',
+  [BlockaidReason.tradeOrderFarming]: 'blockaidDescriptionMightLoseAssets',
+  [BlockaidReason.unfairTrade]: 'blockaidDescriptionMightLoseAssets',
+
+  [BlockaidReason.rawNativeTokenTransfer]: 'blockaidDescriptionTransferFarming',
+  [BlockaidReason.transferFarming]: 'blockaidDescriptionTransferFarming',
+  [BlockaidReason.transferFromFarming]: 'blockaidDescriptionTransferFarming',
+
+  [BlockaidReason.other]: 'blockaidDescriptionMightLoseAssets',
+});
+
+/** List of suspicious reason(s). Other reasons will be deemed as deceptive. */
+const SUSPCIOUS_REASON = [BlockaidReason.seaportFarming];
```

Comment:
The only suspicious reason is rawSignatureFarming from [this comment](https://github.com/metamask/metamask-planning/issues/396#issuecomment-1642092127)
```suggestion
const SUSPCIOUS_REASON = [BlockaidReason.rawSignatureFarming];
```

Reply:
thank you! fixed [befa620](https://github.com/MetaMask/metamask-extension/pull/20051/commits/befa620bbaa803f8868d2987a0689a6eb78818b1)

bug happened here: https://github.com/MetaMask/metamask-extension/pull/20051/commits/dcfa47a651b4b785aa7e482ed2a7a33e5f33d39d